.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: go
 
 go:
-  - 1.9.x
+  - 1.10.x
```

Comment:
Why do you need to bump go version?

Reply:
`DisallowUnknownFields` is introduced in golang 1.10.

Search `encoding/json` in the golang 1.10 [release notes](https://golang.org/doc/go1.10#minor_library_changes), you will see.